### ISSUE TITLE
How two rewrite sg_context and sg_conv in pytorch?

### ISSUE THREAD

**Issue by pipisjy** [nyummvc/CPD-Net#3]
hi!
i want to add some feature processing module follow your feature descriptors in pytorch, and i foud it's hard to sovle this problem.
Could you give me some advice ? 

and i have another question 
v2=v1.sg_conv(dim=64, size=(1,1), name='gen1',pad="SAME",bn=True)

conv2 = torch.nn.conv2d(16,64,kernel_size=(1,1), bias=True, padding=0)
bn2 = nn.BatchNorm2d(64)
bn2(conv2(v1)

are these two equal ?
thank you
